src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1396 @@
+        DS_ASSERT(ret == dsERR_NONE);
+
+        /* Register AV latency changes callback */
+        UT_LOG_INFO("Calling dsHdmiInRegisterAVLatencyChangeCB(IN:CBFun:[0x%0X])", hdmiInAVLatencyChangeCB);
+        ret = dsHdmiInRegisterAVLatencyChangeCB(hdmiInAVLatencyChangeCB);
+        UT_LOG_INFO("Result dsHdmiInRegisterAVLatencyChangeCB(IN:CBFun:[0x%0X]) dsError_t:[%s]",
+                            hdmiInAVLatencyChangeCB, UT_Control_GetMapString(dsError_mapTable, ret));
+        DS_ASSERT(ret == dsERR_NONE);
+
+        /* Register AVI Content changes callback */
+        UT_LOG_INFO("Calling dsHdmiInRegisterAviContentTypeChangeCB(IN:CBFun:[0x%0X])", hdmiInAviContentTypeChangeCB);
+        ret = dsHdmiInRegisterAviContentTypeChangeCB(hdmiInAviContentTypeChangeCB);
+        UT_LOG_INFO("Result dsHdmiInRegisterAviContentTypeChangeCB(IN:CBFun:[0x%0X]) dsError_t:[%s]",
+                            hdmiInAviContentTypeChangeCB, UT_Control_GetMapString(dsError_mapTable, ret));
+        DS_ASSERT(ret == dsERR_NONE);
+    }
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test get number of input ports.
+*
+* This test function gets HdmiInput ports on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 002@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_inputports(void)
+{
+    gTestID = 2;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    uint8_t numInputs = 0;
+    dsError_t ret   = dsERR_NONE;
+
+    UT_LOG_INFO("Calling dsHdmiInGetNumberOfInputs(OUT:numInputs:[ ])");
+
+    ret = dsHdmiInGetNumberOfInputs(&numInputs);
+
+    UT_LOG_INFO("Result dsHdmiInGetNumberOfInputs(OUT:numInputs:[%d])",numInputs);
+    DS_ASSERT(ret == dsERR_NONE);
+
+    if(!numInputs)
+       UT_LOG_INFO("Result: Platform does not supports HdmiIn Ports");
+
+    UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test get number of input ports.
+*
+* This test function gets HdmiInput ports on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 003@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_status(void)
+{
+    gTestID = 3;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    dsHdmiInStatus_t inputstatus = {0};
+    
+    UT_LOG_INFO("Calling dsHdmiInGetStatus(OUT:dsHdmiInStatus_t:[])");
+
+    ret = dsHdmiInGetStatus(&inputstatus);
+
+    UT_LOG_INFO("Result dsHdmiInGetStatus(OUT:inputstatus:[isPresented:[%s], activeport:[%s] ])",
+                 UT_Control_GetMapString(bool_mapTable, inputstatus.isPresented),
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, inputstatus.activePort));
+    DS_ASSERT(ret == dsERR_NONE);
+
+    for(int i = 0 ; i < dsHDMI_IN_PORT_MAX ; i++) {
```

Comment:
As per coding standard, '{' must be place in new line.

Please check it once this case throughout the file once.

Reply:
done